python/requirements/ml/core-requirements.txt

```diff
@@ -8,7 +8,7 @@ xgboost==1.7.6
 lightgbm==3.3.5
 
 # Huggingface
-transformers==4.36.2
+transformers==4.40.0
```

Comment:
this won't work as it is unfortunately, you'll need to recompile the requirements_compiled.txt (step 4 in https://www.notion.so/anyscale-hq/OSS-Python-dependency-management-f32633b0018c423f927727807ea9da08)

Reply:
does CI still do it and output the compiled version?

Reply:
oh nice it seems it does

Reply:
it is yess, it should be the artifact of this job https://buildkite.com/ray-project/premerge/builds/26290#018f8782-65e9-40ba-ad54-655f6c33f0bc

Reply:
cool trying that